packages/react-components/review/react-components.api.md

```diff
@@ -257,6 +257,7 @@ export const DEFAULT_COMPONENT_ICONS: {
     ControlButtonParticipants: JSX.Element;
     ControlButtonScreenShareStart: JSX.Element;
     ControlButtonScreenShareStop: JSX.Element;
+    CameraSwitch: JSX.Element;
```

Comment:
This name is too generic without the context of where it is used. Imagine a developer looking at all the icons and trying to understand what to set this to!

How about "LocalVideoCameraCycler"?